src/ui/public/vis/vis.js

```diff
@@ -84,11 +81,9 @@ export function VisProvider(Private, Promise, indexPatterns, timefilter, getAppS
 
       this.API = {
         savedObjectsClient: savedObjectsClient,
-        SearchSource: SearchSource,
```

Comment:
I'm having the following error `Uncaught TypeError: kbnApi.SearchSource is not a constructor` using the example flights dashboard. Seems that input controls vis is not refactored after removing  SearchSource from vis API.